src/components/organisms/Table.jsx

```diff
@@ -24,17 +25,47 @@ const Title = styled.h1`
   font-size: 1.4rem;
 `;
 
+const ButtonFixedStyles = styled.div`
+  position: fixed;
+  right: 50px;
+  top: 50px;
+`;
+
 const Table = ({ children, title }) => {
   const handleOnClick = () => {
     alert("not yet supported lol )");
   };
 
+  const handleScroll = () => {
+    window.scrollTo({ top: 0, behavior: "smooth" });
+  };
+
+  const [isHidden, setIsHidden] = useState(true);
+
+  useEffect(() => {
+    document.addEventListener("scroll", () => {
+      if (window.scrollY === 0) {
+        setIsHidden(true);
+        return;
+      }
+
+      setIsHidden(false);
+    });
+
+    return () => {
+      document.removeEventListener("scroll", () => {});
+    };
+  }, []);
+
   return (
     <>
       <Header>
         <Title>{title}</Title>
         <Button name="New Project" $color="#148AF1" handleOnClick={handleOnClick} />
       </Header>
+      <ButtonFixedStyles>
+        <Button name="To top" $color="#ccc" isHidden={isHidden} handleOnClick={handleScroll} />
```

Comment:
It's better to not pass isHidden props to Button and use conditional rendering
```suggestion
        {isHidden && <Button name="To top" $color="#ccc" isHidden={isHidden} handleOnClick={handleScroll} />}
```
This will render Button only then isHidden = true